uber/templates/regform.html

```diff
@@ -89,7 +91,7 @@
 
 {% if c.PAGE == 'confirm' %}
     <div class="form-group">
-        <label for="email" class="col-sm-2 control-label">Legal Name</label>
+        <label for="full_name" class="col-sm-2 control-label">Legal Name</label>
```

Comment:
Good catch.  To be honest I've been very lax about the ``for`` attribute; I don't think I've even bothered to include it in any of the plugins I've written.  Which I really should.  Ah well, good fix.